testcrate/tests/misc.rs

```diff
@@ -92,6 +92,47 @@ fn leading_zeros() {
     })
 }
 
+#[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
+#[test]
+fn bswap() {
+    use compiler_builtins::int::bswap::{__bswapdi2, __bswapsi2};
+    fuzz(N, |x: u32| {
+        let x_std = x.swap_bytes();
+        let x_intr = __bswapsi2(x);
+        if x_std != x_intr {
+            panic!("__bswapsi2({}): std: {}, builtins: {}", x, x_std, x_intr);
+        }
```

Comment:
Just use `assert_eq!`, it also prints the two compared values if it fails

```suggestion
        assert_eq!(x_std, x_intr, "__bswapsi2({x})");
```

Same for the below

Reply:
Done